build.gradle.kts

```diff
@@ -1,11 +1,9 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 plugins {
-    `java-library`
     `maven-publish`
     signing
-    kotlin("jvm") version "1.7.20"
+    kotlin("multiplatform") version "1.7.20"
```

Comment:
Note: this will download about 500MB extra on each CI, and additionally requires 2GB of permanent disk space.

Reply:
`additionally requires 2GB of permanent disk space on local machines.` I think we have gradle caches so it is not a net new addition if the dev machine already compiles kotlin multiplatform?

Reply:
> if the dev machine already compiles kotlin multiplatform

That's why I noticed it, this is the first KMP build I did.

> we have gradle caches

I don't think `~/.konan` is part of Gradle caches, it seems that Kotlin Native doesn't use the traditional Gradle features to download stuff. Anyway, not much we can do if we want to accept this build feature.